onnxruntime/test/providers/cpu/tensor/cast_op_test.cc

```diff
@@ -1484,7 +1484,7 @@ void CastOpTestFloatFloat4(std::vector<int64_t> shape,
   std::vector<F4> fp4_data;
   fp4_data.reserve(num_fp4_elements);
 
-  for (size_t i = 0; i < num_pairs; ++i) {
+  for (int i = 0; i < num_pairs; ++i) {
```

Comment:
why change i to int when num_pairs is size_t?
 That will cause sign and unsigned comparison warning.